src/core/mavlink_parameters.cpp

```diff
@@ -102,6 +123,21 @@ void MAVLinkParameters::get_param_async(
     _work_queue.push_back(new_work);
 }
 
+std::map<std::string, MAVLinkParameters::ParamValue> MAVLinkParameters::get_all_server_params()
+{
+    return _param_server_store;
+}
+
+std::pair<MAVLinkParameters::Result, MAVLinkParameters::ParamValue>
+MAVLinkParameters::get_server_param(const std::string& name, ParamValue value_type)
+{
+    if (_param_server_store.find(name) != _param_server_store.end()) {
+        auto value = _param_server_store.at(name);
+        return {MAVLinkParameters::Result::Success, value};
+    }
+    return {MAVLinkParameters::Result::WrongType, {}};
```

Comment:
Is `WrongType` the right error here, or should we add something like `NotFound`?